# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/esden/netcat/default-unsigned-arch.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


This patch fixes arches like PowerPC which do default to unsigned char.

  - Rene Rebe <rene@rocklinux.org>

--- netcat-0.6.1/src/flagset.c.orig	2002-10-03 12:25:16.000000000 +0200
+++ netcat-0.6.1/src/flagset.c	2004-06-28 23:00:40.529869256 +0200
@@ -134,7 +134,7 @@
 
 int netcat_flag_count(void)
 {
-  register char c;
+  register signed char c;
   register int i;
   int ret = 0;
 
